packages/fxa-payments-server/package.json

```diff
@@ -14,7 +14,7 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "test": "npm-run-all test:*",
-    "test:frontend": "react-scripts test --coverage --verbose",
+    "test:frontend": "react-scripts test --coverage --verbose --transformIgnorePatterns 'node_modules/(?!speed-trap)/'",
```

Comment:
Jest wants this particular module transpiled.  Moreover, adding that option into `jest.config.js` did not have any effect.  😞

Reply:
👍